share/rocm/cmake/ROCMSetupVersion.cmake

```diff
@@ -87,7 +87,7 @@ function(rocm_get_version OUTPUT_VERSION)
     rocm_get_build_info(BUILD_INFO -)
 
     if(GIT)
-        set(GIT_COMMAND ${GIT} describe --dirty --long --match [0-9]*)
```

Comment:
You should leave a comment that we will addthe dirty flag again for 2.8.

Reply:
I think two PRs is best as one fix and another changes the behavior. The dirty removal will probably be reverted.

Reply:
Maybe a branch for rocm 2.7 is best for this fix. RC candidate branch is the best approach going forward.

Reply:
@pfultz2 The reason --dirty is being removed is CI cannot be changed fast enough for rocm 2.7 to prevent the dirty flag being raised. Jenkins distribution of code from master to slaves is the reason.

Reply:
> You should leave a comment that we will addthe dirty flag again for 2.8.

Ack. Addressed.

Reply:
> I think two PRs is best as one fix and another changes the behavior. The dirty removal will probably be reverted.

Ack.

Reply:
> Maybe a branch for rocm 2.7 is best for this fix. RC candidate branch is the best approach going forward.

I think it would be better to put the removal of "dirty" flag in a separate branch for 2.7.